### PR TITLE
python3-msgpack: update to 1.0.5.

### PR DESCRIPTION
**Pull request by mhmdanas**
#### Testing the changes
- I tested the changes in this PR: **briefly**

@skmpz

### CODE REVIEW

File: srcpkgs/python3-msgpack/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
 depends="python3"
 checkdepends="python3-pytest"
-short_desc="Python module for Msgpack serializer (Python3)"
+short_desc="Python module for Msgpack serializer"
 maintainer="skmpz <dem.procopiou@gmail.com>"
 license="Apache-2.0"
-homepage="https://msgpack.org/"
```
Comment (abenson) [inline srcpkgs/python3-msgpack/template:13]: Why change the homepage?

Comment (mhmdanas) [reply srcpkgs/python3-msgpack/template:13]: The original one is a general homepage for msgpack, not for python3-msgpack specifically. If that's not a good enough justification, I'm fine with changing it back.

Comment (abenson) [reply srcpkgs/python3-msgpack/template:13]: That's the homepage listed on the Github page

Comment (mhmdanas) [reply srcpkgs/python3-msgpack/template:13]: Still, when I'm looking for information about python3-msgpack specifically, https://msgpack.org is less than ideal. I don't see a reason to blindly use the homepage listed on the GitHub page.

**Pull request closed without merging**